guides/plugins/apps/shipping-methods.md

```diff
@@ -0,0 +1,276 @@
+
+#### Min / Max
+The min and max values depend on the unit. Assuming the unit is days, in our example, the delivery time has a range from 2 to 4 days.
+
+#### Unit
+The following values are possible units
+- hour
+- day
+- week
+- month
+- year
+
+```xml
+// manifest.xml
+
+<?xml version="1.0" encoding="UTF-8" ?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-2.0.xsd">
+    
+    ...
+
+    <shipping-methods>
+
+        <shipping-method>
+            <identifier>NameOfYourFirstShippingMethod</identifier>
+            <name>First shipping method</name>
+            ...
+            <delivery-time>
+                <id>c8864e36a4d84bd4a16cc31b5953431b</id>
+                <name>From 2 to 4 days</name>
+                <min>2</min>
+                <max>4</max>
+                <unit>day</unit>
+            </delivery-time>
+            ...
+        </shipping-method>
+
+    </shipping-methods>
+</manifest>
+```
+
+### Extended configuration
+The functionality of course offers more than one identifier name. The following examples represents all possible configurations.
+
+* Translation of field that are visible to the customer and requires a translation
+* Shipping method description
+* Shipping method icon
+* Shipping method active (expects true or false). Default value is `false`
+
+```xml
+// manifest.xml
+
+<?xml version="1.0" encoding="UTF-8" ?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-2.0.xsd">
+
+    ...
+
+    <shipping-methods>
+
+        <shipping-method>
+            <!-- Identifier should not change after first release -->
+            <identifier>NameOfYourFirstShippingMethod</identifier>
+            <name>First shipping method</name>
+            <name lang="de-DE">Erste Versandmethode</name>
+            <delivery-time>
+                <id>c8864e36a4d84bd4a16cc31b5953431b</id>
+                <name>From 2 to 4 days</name>
+                <min>2</min>
+                <max>4</max>
+                <unit>day</unit>
+            </delivery-time>
+            <!-- The following configurations are optional -->
+            <description>This is a simple description</description>
+            <description lang="de-DE">Das ist eine einfache Beschreibung</description>
+            <icon>icon.png</icon>
+            <active>true</active>
+            <tracking-url>https://www.yourtrackingurl.com</tracking-url>
+        </shipping-method>
+
+    </shipping-methods>
+</manifest>
+```
+### Description
+You can initially add a description for the customer.
+
+::: info
+Please note that the description cannot be modified by the manifest, as the merchant has the ability to change it.
```

Comment:
it will not be updated. but you can provide it

Reply:
@swDennis would you change it to 
"Please note that the description cannot be modified by the manifest once the app is installed, as the merchant can change it." or similar. I can add it as well :slightly_smiling_face: